apps/console/app/lib/auth/stack-auth.ts

```diff
@@ -52,6 +52,13 @@ const authService = {
     });
     return apiKey.value;
   },
+
+  async getAccessToken() {
+    const user = await getStackApp().getUser();
```

Comment:
StackAuth SDK is poorly written. I'm afraid this will cause performance issues, since it is executed on every route navigation. Instead, manually extract the access token from the cookie, like you did on the server side before.